docs/api/utils.rst

```diff
@@ -0,0 +1,604 @@
+.. include:: header.rst
+
+============
+Functions
+============
+The following are miscellaneous functions and attributes on a fairly low-level technical detail.
+
+Some functions provide detail access to PDF structures. Others are stripped-down, high performance versions of other functions which provide more information.
+
+Yet others are handy, general-purpose utilities.
+
+
+==================================== ==============================================================
+**Function**                         **Short Description**
+==================================== ==============================================================
+:meth:`ConversionHeader`             return header string for *get_text* methods
```

Comment:
When we provide the short description can we just capitilize the first letter in the first word of each one as a rule? 
I see:
`return header string .....`
Then:
`Convenience function ....`
then a mixture ....